src/Question.js

```diff
@@ -10,19 +11,29 @@ class Question extends Component {
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
     })),
-    onClick: PropTypes.func.isRequired
+    onClick: PropTypes.func.isRequired,
+    questionNumber: PropTypes.number.isRequired,
+    userAnswer: PropTypes.bool
+  }
+  getQuestionClassName() {
+    return classNames({
+      question: true,
+      rightAnswer: (this.props.userAnswer && (this.props.answers.correct = true)) ? true : false,  
+      wrongAnswer: (this.props.userAnswer && (this.props.answers.correct = false)) ? true : false 
+    });
```

Comment:
The good news here is that you're making things more complicated than they need to be.

`(this.props.userAnswer && (this.props.answers.correct = true)) ? true : false`

So lets break this down. You've got your statement before the ternary indicator (the ? mark) then the ternary says - if that statement is true evaluate to true, if it's false evaluate to false.... the thing is that the statement before the ? will already evaluate to true or false. So we don't really need the ternary.

`this.props.userAnswer && (this.props.answers.correct = true)`

Now we need to look at this statement. Again, you're almost there. The first part says, "is there an answer?." The second part SHOULD say, "is the answer correct", but what it actually says is "assign this.props.answers.correct as true". This comes down to the difference betwee "=" and "===". A single equals sign (=) assigns a variable a value. The equality check operator (===) checks whether two values are the same and evaluates to either true or false. So in this case we simply need to use ===.